### PR TITLE
TIP-706: Add an "ID" product sorter

### PR DESCRIPTION
**Pull request by damien-carcel**
## Description

- [x] Add sorter integration tests for the field in the PQB
- [ ] ~Add missing integration to the PimCatalogXIntegration tests for sort~ Not needed
- [x] Add the sorter ~+ specs~
- [ ] Update the reference documentation

## Definition Of Done

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | -
| Added Behats                      | -
| Added integration tests           | OK
| Changelog updated                 | -
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | -
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Review by jjanvier — APPROVED** [review]

**Review by skeleton — APPROVED** [review]
thank you :+1:

File: src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst [hunk 1]
```diff
@@ -508,6 +508,100 @@ Descendant
...
+Id
+**
+:Apply: apply datatype 'keyword' on the 'id' field
+
+Data model
+~~~~~~~~~~
+.. code-block:: php
+
+    [
+        'id': 'prct-eb-1256'
```
Comment (SamirBoulil) [inline src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst]: =>

**Review by SamirBoulil — APPROVED** [review]
GTM

File: src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst (hunk 1 above)
Comment (damien-carcel) [reply src/Pim/Bundle/CatalogBundle/Elasticsearch/Filter/search_specification.rst]: Done

**Pull request merged**